charts/linkerd-crds/templates/policy/unmeshed-network.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: unmeshednetworks.policy.linkerd.io
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+  labels:
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    linkerd.io/control-plane-ns: {{.Release.Namespace}}
+spec:
+  group: policy.linkerd.io
+  names:
+    categories:
+    - policy
+    kind: UnmeshedNetwork
+    listKind: UnmeshedNetworkList
+    plural: unmeshednetworks
+    singular: unmeshednetwork
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        description: >-
+          An UnmeshedNetwork captures traffic to destinations considered
+          to be external to the mesh.
+        type: object
+        required: [spec]
+        properties:
+          apiVerson:
+            type: string
+          kind:
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              defaultPolicy:
+                type: string
+                enum:
+                - AllowAll
+                - DenyAll
```

Comment:
* We should find a better name than `defaultPolicy`. We use 'policy' to describe many different types of configuration options. What kind of policy are we enforcing here? As an exercise: let's try to document this field to precisely describe what it does.

Given that this we anticipate this resource being used to discuss both egress and non-egress unmeshed traffic, these values future-proof? Would we ever want to differentiate denial for in-cluster resources from unknown resources? Should this be `AllowUnknown` and `DenyUnknown`? Something else?

Reply:
I have tried to come up with more descriptive names here and have added some additional documentation. LMWYT: https://github.com/linkerd/linkerd2/pull/13150/commits/4e1269bccc42877e1dcc14f774be368c3eb58db7